mln/models/module.py

```diff
@@ -29,6 +29,8 @@ class Meta:
 		constraints = (models.UniqueConstraint(fields=("owner", "pos_x", "pos_y"), name="module_unique_owner_pos"),)
 
 	def __str__(self):
+		if self.pos_x is None or self.pos_y is None:
+			return "%s's %s, %i clicks" % (self.owner, self.item.name, self.total_clicks)
 		return "%s's %s at pos (%i, %i), %i clicks" % (self.owner, self.item.name, self.pos_x, self.pos_y, self.total_clicks)

```

Comment:
This could be fixed more simply by changing `(%i, %i)` to `(%s, %s)` which also can also format `None`.